cns/restserver/ipam.go

```diff
@@ -192,6 +190,34 @@ func (service *HTTPRestService) requestIPConfigsHandler(w http.ResponseWriter, r
 	logger.ResponseEx(service.Name+operationName, ipconfigsRequest, ipConfigsResp, ipConfigsResp.Response.ReturnCode, err)
 }
 
+func (service *HTTPRestService) updatePodInfoWithInterfaces(ipconfigResponse *cns.IPConfigsResponse) (*cns.IPConfigsResponse, error) {
+	for _, podIpInfo := range ipconfigResponse.PodIPInfo {
+		hostPrimaryInterface, err := service.getPrimaryHostInterface(context.TODO())
```

Comment:
You should pass down the context from `requestIPConfigsHandler`

Reply:
updated to pass`r.Context()` from `requestIPConfigsHandler`